src/Components/CartCard/CartCard.js

```diff
@@ -25,24 +29,44 @@ export default function CartCard() {
                 <div className="description1">
                   <p className="item-price">₹ {product.price}</p>
                   <p>
-                    <label htmlFor="quantity">Quantity:</label>
-                    <input
-                      className="itemQuantity"
-                      type="number"
-                      id="quantity"
-                      name="quantity"
-                      min="1"
-                      max="5"
-                      value={product.quantity}
-                    />
+                    <label htmlFor="quantity">
+                      Quantity:{" "}
+                      <button
+                        className="btn-changeQty"
+                        onClick={() =>
+                          dispatch({
+                            type: "REDUCE",
```

Comment:
You can use a better type name like: reduce cart quantity.

Reply:
noted

Reply:
done